integration/container/exec_test.go

```diff
@@ -117,3 +117,18 @@ func TestExec(t *testing.T) {
 	assert.Assert(t, is.Contains(out, "PWD=/tmp"), "exec command not running in expected /tmp working directory")
 	assert.Assert(t, is.Contains(out, "FOO=BAR"), "exec command not running with expected environment variable FOO")
 }
+
+func TestExecUser(t *testing.T) {
+	skip.If(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.35"), "broken in earlier versions")
```

Comment:
oh, actually; we should adjust this version; I can backport this to 18.09 (so then it'll be `1.39`)

Reply:
What would that matter?

Reply:
(I _think_) these tests are still integrated in the internal e2e test-suite, where they run with different combinations of daemons; making sure that the version in which it works matches.